konfidency/utils/types.ts

```diff
@@ -42,3 +42,27 @@ export interface Composition {
 export interface CustomerSettings {
   minStarsHighlightPDP: number;
 }
+
+export interface WriteReview {
+  userId: string;
+  email: string;
+  rating: number;
+  text: string;
+  recommended: boolean;
```

Comment:
```suggestion
  recommended?: boolean;
```
by the doc example this prop is not required